app/Models/Post.php

```diff
@@ -33,11 +33,11 @@ public function content()
     }
 
     /**
-     * Each post belongs to an event.
+     * Each post has events.
      */
-    public function event()
+    public function events()
     {
-        return $this->belongsTo(Event::class);
+        $this->morphMany('Rogue\Models\Event', 'eventable');
```

Comment:
Let's use [::class](http://php.net/manual/en/language.oop5.basic.php#language.oop5.basic.class.class) for all of these relationships: `$this->morphMany(Event::class, 'eventable');`